### DISCUSSION TITLE
Cookies essentiels et optionnels

### DISCUSSION THREAD

**Discussion by KillyanBtllr** [AmauriC/tarteaucitron.js#1285] [Q&A] [answered]
Bonjour,

Je suis en train de tester tarteaucitron et je voulais savoir si il était possible de différencier les cookies essentiels et les optionnels ?
Dans mon cas, j'aimerais ajouter un service et j'aimerais le rendre "obligatoire".

Merci d'avance

**Comment by AmauriC** [comment]
Bonjour Killyan,

tarteaucitron gère le consentement, si vous voulez qu'un service ne soit pas soumis à consentement, il suffit de ne pas l'inclure dans tarteaucitron ; )

**Reply by KillyanBtllr** [reply]
Effectivement mais l'utilisateur ne pourra donc pas voir ces cookies listés dans l'onglet de tarteaucitron pour l'informer des cookies dit essentiels ?

**Reply by AmauriC** [reply, marked as answer]
Il y a l'option "afficher un message pour les cookies obligatoires" et vous pouvez modifier le texte pour décrire pourquoi Matomo n'est pas soumis à consentement.

**Reply by KillyanBtllr** [reply]
Ah oui mince, je n'avais pas fais attention aux options mandatory ! 
Je suppose que la modification du texte mandatory se fait avec la variable tarteaucitronCustomText ?
